ush/setup.sh

```diff
@@ -811,6 +816,7 @@ FIX_SMOKE_DUST=${FIX_SMOKE_DUST:-"${HOMErrfs}/fix/smoke_dust"}
 FIX_BUFRSND=${FIX_BUFRSND:-"${HOMErrfs}/fix/bufrsnd"}
 AIRCRAFT_REJECT=${AIRCRAFT_REJECT:-"${FIX_GSI}"}
 SFCOBS_USELIST=${SFCOBS_USELIST:-"${FIX_GSI}"}
+FIX_MINMAXTRH=${FIX_MINMAXTRH:-"${HOMErrfs}/fix/minmaxtrh"}
```

Comment:
Could you make a copy of FIX_MINMAXTRH to Hera or Jet?  I can facilitate syncing them to all available rrfs-workflow platforms.

Reply:
We need to sync fix files in all rrfs-workflow platforms.